### ISSUE TITLE
<style src="./style.scss"></style> doesn't work on svelte 3

### ISSUE THREAD

**Issue by evnix** [kaisermann/svelte-preprocess#50]
I tried following the docs, here is what I have,
https://codesandbox.io/s/exciting-sound-85w5e

it gives no error, but doesn't produce any style either

**Comment by kaisermann** [comment]
I think I should've made it more clear on the docs that external files support is only available when using `svelte-preprocess` in the `auto process` mode: https://github.com/kaisermann/svelte-preprocess#external-files-support

Anyway, thanks for the feedback! I'll probably end up adding support for importing external sources for script and style stand-alone processors.

**Comment by kaisermann** [comment]
Added support for external srcs for stand-alone processors on `v2.15.0` 😁

**Comment by evnix** [comment]
@kaisermann Thanks a lot for the quick turn around, but still doesn't work with `v2.15.0`
as you can see no style has been applied to hello world page.
I am not sure if I am missing something else.
If you can tell me what it is or you can fork it right there, we can use that as a live example which people can straight away copy things from, as I found it very hard to follow the docs.

**Comment by kaisermann** [comment]
@evnix Can you try exporting the codesandbox project as a zip and running it locally? I just did that and it seems to be working fine. It has probably something to do with using `fs` methods inside the sandbox environment 🤔

**Comment by evnix** [comment]
Thanks a lot, that worked!
As a suggestion for docs, In the **Usage** section, The "With rollup-plugin-svelte" should be at the top as most developers who start with svelte will only have this file and is a good place to start. 

It took me quite a lot of time to figure out that I could completely skip the "Auto Preprocessing" , "Basic", "Advanced", "Stand-alone processors" section as I was wondering for a lot of time where do I put the code from those sections.

**Comment by kaisermann** [comment]
@evnix Thanks for the feedback! I just rearranged a some sections of the readme and moved the usage with bundlers/other tools up on the document 😁
